node/src/neighborhood/mod.rs

```diff
@@ -800,12 +804,28 @@ impl Neighborhood {
             payload_size: 10000,
         };
         if self.handle_route_query_message(msg).is_some() {
-            self.overall_connection_status.update_can_make_routes(true);
+            debug!(
+                &self.logger,
+                "The check_connectedness() found a 3 hops route for the first time."
```

Comment:
"The check_connectedness()" is a bit clumsy. Maybe "The connectivity check?" Also, and maybe more importantly, I'm not sure "for the first time" will always be correct. What if a Node gets fully connected, then loses enough neighbors to go back to just connected-to-neighbor, then gains neighbors again and goes back to full connectivity? Will this message ("for the first time") get fired again? If so, it'll be misleading the second time.